rook/README.md

```diff
@@ -0,0 +1,39 @@
+# Rook installation
+
+Also see the official documentation at <https://rook.io/docs/rook/v1.5/> and the repository at <https://github.com/rook/rook>.
+
+## Quickstart
+
+```bash
+KUBECONFIG=/path/to/kubeconfig ./deploy-rook.sh
+```
+
+## Requirements
+
+### Ceph partition or disk
+
+An empty disk or partition needs to be available.
+Ceph will create an [OSD](https://docs.ceph.com/en/latest/man/8/ceph-osd/) for each partition or disk to provide access to it.
+
+To create an empty partition `/dev/vda2` on the `/dev/vda` disk, leaving 50 gb to other partitions, use the following cloud-init config:
```

Comment:
Nit-pick: Please capitalize "GB" (and teach me how to make suggestions on GitHub). :smile:

Reply:
When you create a comment in github it's the first button to the right of `write` and `preview`.